autonomy/cli/scaffold_fsm.py

```diff
@@ -416,6 +416,10 @@ def _update_models(  # pylint: disable=no-self-use
         config.models.create(
             "abci_dialogues", SkillComponentConfiguration("AbciDialogues")
         )
+        config.models.create(
+            "benchmark_tool",
+            SkillComponentConfiguration("BenchmarkTool", log_dir="/logs"),
```

Comment:
`/` will cause issue on windows?

Reply:
All the skills have the `/logs` in their skill.yaml so I don't think it should be an issue.